skbase/base/_base.py

```diff
@@ -814,6 +814,11 @@ class BaseEstimator(BaseObject):
     Extends BaseObject to include basic functionality for fittable estimators.
     """
 
+    # tuple of non-BaseObject classes that count as nested objects
+    # get_fitted_params will retrieve parameters from these, too
+    # override in descendant class - common choice: BaseEstimator from sklearn
+    GET_FITTED_PARAMS_NESTING = ()
```

Comment:
Why is this not a tag?

Reply:
it's more a config for the entire package than a tag that is estimator specific.
Not sure what the best way is - I changed it from being `sklearn` `BaseEstimator` (hard-coded) in `sktime`.

Reply:
Aren't we using tags as things set by developers, and configs more things set by end-users? This would seem like something that is set once by the developer using `skbase`, instead of an end-user. But I do agree, it is more of a package level thing. But maybe end users would want to turn it off (not sure, but who knows)?

Let's keep this in mind when we work on the interface for package level config.

Reply:
yes, you are right!
It is a developer thing, and it might be worth setting at the estimator level.

I now start agreeing witih you that it probably should be a tag.

The one question I still have then is, there are currently no tags in `skbase`. This would open a can of worms of "tags already defined by `skbase` (although the trade-off is to explaining that attribute).